docs/configuration.md

```diff
@@ -12,6 +12,7 @@
 | scan.identifier.regex | Regex that will be used to determine an identifier for each file. It has to contain a capturing group. Identifiers are then used to sort the scripts before running them. | ```(.*)``` (with this regex, the script's whole name will be its identifier) | no | With the regex ```(.*?)_.*```, a script named ```1.23_my-script.js``` will have ```1.23``` as its identifier  |
 | scan.tags.createFromFolder | If true, scripts will have their parent folders names as tags. Relative path to ```scan.path``` is used.  | ```false``` | no | ```false``` or ```true``` |
 | tag.*your_tag* | Glob paths to your scripts that you want to apply the tag "your_tag" on. To declare multiple tags, you will have to add multiple properties in your settings. A tag ```my_tag``` will have as as property name ```tag.my_tag``` **WARNING:** ALWAYS declare your tags using absolute paths. Relative paths and even using a tilde (~) won't do the trick. |  | no | ```[data/*, script1.js, old/old_script1.js]``` |
+| filter.filename.regex | Scripts that have a filename matching the regex will be considered | ```(.*)``` | yes | ```(.*)(\.js$)``` |
```

Comment:
I find the groups confusing because there is no group extraction
```suggestion
| filter.filename.regex | Scripts that have a filename matching the regex will be considered | ```.*``` | yes | ```.*\.js$``` |
```